lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -60,7 +60,6 @@ llvm::Error setTensorType(const caffe2::TensorProto &in, Tensor *T) {
     }
     dim.push_back(d);
   }
-
```

Comment:
sorry, not your change, but
line 46 does not have a correct comment.
Could you fix [-128, 127] and <=255.

Reply:
I wont bother having one more line of credit lol